STYLE_GUIDE.md

```diff
@@ -0,0 +1,147 @@
+# Style Guide
+
+## Color Palette
+
+**Monochromatic Theme Colors**
+
+$theme-colors: (
+    PRIMARY: **#252525**            // DARK GREY
+    ACCENT: **05C3F9**              // SKY BLUE
+    SECONDARY: **#383838**          // LIGHT GREY
+    BASE: **#FFFFFF**               // WHITE ------ DOCUMENT BACKGROUND
+);
```

Comment:
Kudos for choosing Markdown for your documentation, and I love the numerous code samples, but you should leverage the [code formatting features](https://help.github.com/articles/markdown-basics/#code-formatting) of Markdown to make those samples easier to read.